source/getting-started/index.rst

```diff
@@ -1,131 +1,411 @@
+   using.
 
-#. In the sidebar, click *Reservations*, then click *Leases*
-#. Click on the *+ Create Lease* button in the toolbar
-#. Type *my_first_lease* for the lease name
-#. Find the *Resource Properties* section. In the dropdown below *node_type*, select *compute_skylake*
-#. Click the *Create* button
+Our preferred node type is available on demand, so let's navigate back to our
+Leases ("Reservations -> Leases") page to create a new lease. Click the "Create
+Lease" button in the top right corner of the page.
 
-.. figure:: create_lease.png
-  :alt: The Create Lease dialog
-  :figclass: screenshot
+.. figure:: ../_static/imgs/getting_started/create-lease-button.png
 
-  The Create Lease dialog - be sure to select compute_skylake in the dropdown below node_type
+Clicking this button will then open a web form. Let's go through the web form
+step by step.
 
-The reservation will start shortly, at which point you can launch an instance on
-a bare metal node.
+**General**
+
+.. figure:: ../_static/imgs/getting_started/create-lease-form-general.png
+   :figwidth: 80 %
+   :align: center
+
+   Specify your lease name and duration.
+
+In this section, add a name for your lease (`my-first-lease`). To
+create an on-demand lease, we can click next, because the form will
+auto-populate with defaults for the duration of the lease (the default is a
+one-day lease that starts immediately). If you want to change the duration of
+your lease or to make an advanced reservation, you can input the start date and
+time, number of days (maximum 7 days), and the end time.
+
+**Hosts**
+
+.. figure:: ../_static/imgs/getting_started/create-lease-form-hosts.png
+   :figwidth: 80 %
+   :align: center
+
+   Specify the number and type of host.
+
+On the next section, you can specify the hardware that you want to
+include in your lease. You must check the box "Reserve Hosts" and fill out the
+required fields. We will start with just one node and will set the minimum and
+maximum number of hosts to 1. In the Resource Property field, we can use
+different attributes of Chameleon resources (such as "node type") to specify
+the exact kind of hardware we want to reserve with this lease. We can add
+multiple fitlers with different properties, but we only care about the node
+type right now.
+
+.. important::
+   If you specify resource properties that return more than one node matching the filter that is available for your specified duration, the system will automatically select a node for you. If you want to specify the precise node that you want to use, you will need to refer to the Node ID and use the Resource Property filter to specify that node ID.
+
+**Networks**
+
+.. figure:: ../_static/imgs/getting_started/create-lease-form-networks.png
+   :figwidth: 80 %
+   :align: center
+
+   Finally, select your network reservation options.
+
+On the final section of the lease form, you can reserve network
```

Comment:
We should say that network reservations are not required for most users, as it's been a point of confusion before.